cozy/view_model/library_view_model.py

```diff
@@ -78,9 +78,9 @@ def selected_filter(self, value):
         self._notify("selected_filter")
 
     @property
-    def is_any_book_in_progress(self) -> bool:
+    def is_any_book_recent(self) -> bool:
         for book in self.books:
-            if book.position > 0:
+            if book.last_played > 0:
```

Comment:
i think finished books should stay on the recent screen,

i have a few books i keep on there because i like to go back to older chapters after finishing or just plain forgot to set a sleep timer and the book finished without me.

if nothing else, "recents" implies any book i opened "recently" regardless of play status should be there, "currently reading" would fit better as a name imo

Reply:
I don't have a strong opinion for this situation. It makes sense for me that you can access your book from the recents screen even if it is finished because you might forget the sleep timer etc.